DataScience/ds_parse.py

```diff
@@ -146,6 +146,8 @@ def json_cooked(x, do_devType=False, do_VWState=False, do_p_vec=False):
     ind6 = x.find(b'"',ind5)
     ind7 = x.find(b',"a"',ind5)
     ind8 = x.find(b']',ind7+7)          # equal to: x.find('],"c',ind7+8)
+    if ind8 == -1:
```

Comment:
when will it run into this case?

Reply:
Probably bad formatted data. Not sure though. I pulled this fix in from master.